docs/modules/device-lab-management/pages/ios-devices/add-ios-device.adoc

```diff
@@ -0,0 +1,318 @@
+
+image::device-lab-management:device-lab-management-add-android-screen-changes-to-blue.PNG[width=300, alt="device screen changes and shows Kobiton name and logo"]
+
+=== iOS 17 and above
+
+[IMPORTANT]
+.Note for air-gapped Mac mini hosts (no Internet access)
+====
+To control the iOS devices, deviceConnect needs to mount a *Developer Disk Image* (DDI), which is a `.dmg` archive included with Xcode that contains executables and other files needed by Xcode to support debugging and testing on iOS devices.
+
+For iOS 17 and later, rather than Xcode providing a different DDI for every iOS version and device architecture, there is a generic DDI that Xcode must "personalize" for each device. The personalization process requires an Internet connection, as Xcode must use Apple's notarization servers to sign the personalized image. Without an Internet connection, Xcode can't personalize a DDI.
+
+If the Mac mini host does not have Internet connection, follow the section xref:#preload-ddi-air-gapped[preload DDI for air-gapped Mac mini] before continuing with this section.
+
+====
+
+Follow the appropriate steps based on whether you are using Standard or Lightning mode
+
+[tabs]
+====
+
+Standard Mode::
++
+--
+
+Open Xcode on the Mac mini host, then navigate to **Window → Devices and Simulators**. Do this before continuing to the next step.
+
+The Trust *this computer* prompt on the device screen appears, tap **Trust**.
+
+image::device-lab-management:device-lab-management-ios-add-ios-standard-lightning-trust.PNG[width=300,alt="Standard Mode. Trust This Computer popup, clicking Trust"]
+
+The Trust *this computer* prompts will reappear, tap **Trust** again. This time there should be no more **Trust** prompts.
+
+--
+
+Lightning mode::
++
+--
+
+Open Xcode on the Mac mini host, then navigate to **Window → Devices and Simulators**. Do this before continuing to the next step.
+
+Make sure you connect the device **to the Mac mini host** first.
+
+The Trust *this computer* prompt on the device screen appears, tap **Trust**.
+
+image::device-lab-management:device-lab-management-ios-add-ios-standard-lightning-trust.PNG[width=300,alt="Lightning Mode. Trust This Computer popup, clicking Trust"]
+
+In the Mac mini host’s screen, under the **Devices** tab of the **Devices and Simulators** screen, the iOS 17 devices should show up with a yellow warning message like the one below:
+
+image::device-lab-management:device-lab-management-ios-add-ios-device-and-simulator.PNG[width=600,alt="iOS 17 in Devices and Simulators"]
+
+Unplug the device from the Mac mini host and plug it into the GEM.
+
+The *Trust this computer* prompts will reappear, tap **Trust** again. This time there should be no more **Trust** prompts.
+
+In Xcode’s Devices and Simulators, the iOS 17 devices now display with a globe icon next to it like below:

```

Comment:
"the iOS 17 devices will now have a globe icon like the below:"

Reply:
updated